### PR TITLE
refactor: use qbx lib

### PR DESCRIPTION
**Pull request by solareon**
## Description

Use qbx lib because utils was bad. This resource still has quite a few issues that need to be addresses outside of just using lib.

## Checklist

- [ ] I have personally loaded this code into an updated Qbox project and checked all of its functionality.
- [x] My pull request fits the contribution guidelines & code conventions.

### CODE REVIEW

**Review by BerkieBb — APPROVED** [review]

File: client/main.lua [hunk 1]
```diff
@@ -326,39 +325,39 @@ RegisterNetEvent('iens:repaira', function()
...
 	if not isPedDrivingAVehicle() then
-		exports.qbx_core:Notify(Lang:t("error.inside_veh_req"))
+		exports.qbx_core:Notify(locale("error.inside_veh_req"))
 		return
 	end
 	vehicle = cache.vehicle
 	if isNearMechanic() then return end
 	if GetVehicleEngineHealth(vehicle) >= cfg.cascadingFailureThreshold + 5 then
-		exports.qbx_core:Notify(Lang:t(('nofix_message_%s'):format(noFixMessagePos)))
+		exports.qbx_core:Notify(locale(('error.nofix_message_%s'):format(noFixMessagePos)))
```
Comment (solareon) [inline client/main.lua:343]: Yeah....this isn't great

File: client/main.lua [hunk 2]
```diff
@@ -326,39 +325,39 @@ RegisterNetEvent('iens:repaira', function()
...
+
 	SetVehicleUndriveable(vehicle, false)
 	SetVehicleEngineHealth(vehicle, cfg.cascadingFailureThreshold + 5)
 	SetVehiclePetrolTankHealth(vehicle, 750.0)
 	healthEngineLast = cfg.cascadingFailureThreshold + 5
 	healthPetrolTankLast = 750.0
 	SetVehicleEngineOn(vehicle, true, false )
 	SetVehicleOilLevel(vehicle, (GetVehicleOilLevel(vehicle) / 3) - 0.5)
-	exports.qbx_core:Notify(Lang:t(('fix_message_%s'):format(fixMessagePos)))
+	exports.qbx_core:Notify(locale(('success.fix_message_%s'):format(fixMessagePos)))
```
Comment (solareon) [inline client/main.lua:360]: Yeah....this isn't great either

**Review by FjamZoo — DISMISSED** [review]
@solareon 

I see a lot of places using double quotes and some places using singular quotes.

To keep resource similarity change them to singular quotes

**Review by BerkieBb — APPROVED** [review]

**Review by Manason — APPROVED** [review]

**Pull request merged**